tests/test_storageclass_deletion.py

```diff
@@ -0,0 +1,221 @@
+    Remove the resources after execution of tests
+    """
+    assert delete_storageclass(self.data['sc_name']), \
+        "Deletion of rbd storage class failed"
+    assert delete_namespace(self.data['project_name']), \
+        "Deletion of namespace failed"
+    utils.delete_file(TEMP_PVC_YAML_FILE)
+    utils.delete_file(TEMP_SC_YAML_FILE)
+
+
+def setup(self):
+    assert create_namespace(**self.data)
+    assert create_storageclass(**self.data)
+
+
+def create_namespace(**kwargs):
+    """
+    Creates a project if it is not already available
+    Args:
+        **kwargs:
+    Returns:
+        (bool): True namespace creation is successful,
+                False if the namespace creation is not successfull.
+    """
+    project_name = (kwargs['project_name'])
+    project_get = NAMESPACE.get()
+    namespaces = [item['metadata']['name'] for item in project_get['items']]
+    log.info(f'checking if project {project_name} already exists')
+    if project_name in namespaces:
+        log.info(
+            f'project {project_name} exists, using the existing namespace'
+        )
+        return True
+    else:
+        log.info(f'creating a new project {project_name}')
+        return run_cmd(f'oc new-project {project_name}')
+
+
+def delete_namespace(project_name):
+    """
+    Deletes the project
+    Args:
+        project_name (str): Project to be deleted
+    Returns:
+        (bool): True if deletion is successful, False otherwise
+    """
+
+    return NAMESPACE.delete(resource_name=project_name)
+
+
+def create_storageclass(**kwargs):
+    """
+    Creates a storage class
+    Args:
+        **kwargs:
+    Returns:
+        (bool): True if creation is successful, False otherwise
+    """
+
+    sc_name = (kwargs['sc_name'])
+    sc_get = SC.get()
+    storage_classes = [item['metadata']['name'] for item in sc_get['items']]
+    log.info(f'checking if {sc_name} exists already')
+    if sc_name in storage_classes:
+        log.info(f'storage class {sc_name} exists, using {sc_name} ')
+        return True
+    else:
+        file_sc = templating.generate_yaml_from_jinja2_template_with_data(
+            SC_YAML, **kwargs
+        )
+        with open(TEMP_SC_YAML_FILE, 'w') as yaml_file:
+            yaml.dump(file_sc, yaml_file, default_flow_style=False)
+        return SC.create(yaml_file=TEMP_SC_YAML_FILE)
+
+
+def delete_storageclass(sc_name):
+    """
+    Deletes a storage class
+    Args:
+        sc_name(str): Storage class to be deleted
+    Returns:
+        (bool): True if deletion is successful, False otherwise
+    """
+
+    log.info(f'deleting storage class: {sc_name}')
+    return SC.delete(resource_name=sc_name)
+
+
+def create_pvc(**kwargs):
+    """
+    Creates a Persistent Volume Claim
+    Args:
+        **kwargs:
+    Returns:
+        (bool): True if creation is successful,
+                False Creation is unsuccessful
+    """
+
+    file_pvc = templating.generate_yaml_from_jinja2_template_with_data(
```

Comment:
not needed since you are loading from defaults.PVC_DICT

Reply:
removed